model_to_project_task/README.rst

```diff
@@ -0,0 +1,66 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Model to Project Task
+=====================
+
+This module allows to end users to create an task from any configured models
```

Comment:
Hi @bealdav thanks for your contribution
I will check mostly format and guidelines in your PR

This module allows end users to create a task from any configured models
and automatically link it to the initial object via a reference field.